src/app/space/create/deployments/resource-usage/loading-utilization-bar.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="progress-container progress-description-left progress-label-right">
+    <div class="progress-description">
+      {{ resourceTitle }} ({{ resourceUnit }})
+    </div>
+    <div class="progress">
+      <div class="progress-bar" role="progressbar" [attr.aria-valuenow]="0" aria-valuemin="0" aria-valuemax="100"
+        [ngClass]="getUtilizationClass()"
+        [style.width]="0 + '%'" data-toggle="tooltip" title="Loading">
```

Comment:
Does the style.width really need to be computed from an expression as a number concatenated with a string? Why not just give it a string literal?